src/Webapi/Webapi__FileList.res

```diff
@@ -0,0 +1,9 @@
+type t = {
+  length: int
+}
+
+@val external toArray: t => array<Webapi__File.t> = "Array.prototype.slice.call"
```

Comment:
Here and there they mention that these list type objects will become arrays but so far they haven't done that in the browsers. I'm guessing it would break to much of the web however I wonder if the bindings should enforce arrays but that would be a bit more like normalization rather than raw bindings?

Reply:
>I wonder if the bindings should enforce arrays but that would be a bit more like normalization rather than raw bindings?

I like the ergonomics of that, but you'd need a wrapper over

```
@get external files: t => Webapi__FileList.t = "files"
```

 to coerce it to an actual array. It could also cause runtime-unsafe code if users start declaring stuff to be `Webapi.FileList.t` and expect to be able to use `Js.Array` functions on it.

Reply:
Right good point. I guess most people will probably convert it to an array to do further processing on the items and it's doubtful that they change these things to arrays without breaking the web.

Reply:
As long as we offer the methods to access the data without converting to an Array - which we have - I think there's no need to do coercion, just offer the `toArray` binding.